omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -2446,7 +2438,7 @@
                                                         </div>
                                                         <div>
                                                             <label>CD4%</label>
-                                                            <obs id="viral-load-results-qualitative" conceptId="730"
+                                                            <obs conceptId="730"
```

Comment:
@slubwama Where are the numeric ID values for the actual viral load values when detected

Reply:
@ssmusoke this was wrongly placed on the CD4 result set so Its being removed to catch potential wrong validations